python/cuml/dask/datasets/regression.py

```diff
@@ -167,6 +326,13 @@ def make_regression(n_samples=100, n_features=100, n_informative=10,
         for reproducible output across multiple function calls.
     n_parts : int, optional (default=1)
         The number of parts of work.
+    order : str, optional (default='F')
+        Row-major or Col-major
+    dtype: str, optional (default='float32')
+        dtype of generated data
+    use_full_low_rank : boolean (default=True)
+        Whether to use the entire dataset to generate the low rank matrix.
+        If False, it uses the first chunk
```

Comment:
We should add a few words here about this because the only difference is not quite that it's only using the first chunk. I would add something along the lines of `If False, it creates a low rank on a single chunk and uses the corresponding covariance to generate a multivariate normal distribution on the remaining chunks.`